.goreleaser.yml

```diff
@@ -1,5 +1,6 @@
 builds:
-  - id: admin-revoker
+  - &build
+    id: admin-revoker
```

Comment:
It's a little scary that the anchor here has real values in it; if one of the entries below failed to overwrite one of the values from the anchor, you might end up with a mismatched "id: akamai-purger; binary: admin-revoker" on accident. Can this file use a fake stanza (like the `x-function` example in the blog Samantha linked) to declare just the fields that we actually want to re-use?

Reply:
I'll look at the rest of the config language and see if i can hide it somewhere.

Reply:
I wasn't able to find a place to create the fake stanza.

Reply:
Oh well, no problem then.